doc/CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+# Sonata project contribution
+
+Thanks for you interest onto Sonata projects!
+
+## Summary
+
+* [Issues]()
+* [Pull Requests](#pull-requests)
+* [Label rules]()
+
+## Pull Requests
+
+All the sonata team will be glad to review your code changes propositions! :smile:
+
+But please, read the following before.
+
+### Coding style
+
+Each project follows [PSR-1](http://www.php-fig.org/psr/psr-1/), [PSR-2](http://www.php-fig.org/psr/psr-2/)
+and [Symfony Coding Standards](http://symfony.com/doc/current/contributing/code/standards.html) for coding style,
+[PSR-4](http://www.php-fig.org/psr/psr-4/) for autoloading.
+
+Please [install PHP Coding Standard Fixer](http://cs.sensiolabs.org/#installation)
+and run this command before committing your modifications:
+
+```bash
+php-cs-fixer fix --verbose
+```
+
+### Writing a Pull Requests
```

Comment:
Fall into the same case as you, and ended up making two PRs, telling to please merge the first one, so that I can rebase the other one on the result (which might not be needed), but it did feel a bit cumbersome. Anyway, we are discussing that with @Soullivaneuh in Soullivaneuh#3 , so please add any further comments about this there.